src/legacy/core_plugins/kibana/public/home/components/welcome.tsx

```diff
@@ -137,21 +112,34 @@ export class Welcome extends React.PureComponent<Props, State> {
           <div className="homWelcome__content homWelcome-body">
             <EuiFlexGroup gutterSize="l">
               <EuiFlexItem>
-                {shouldShowTelemetryOptIn && step === 0 && (
-                  <TelemetryOptInCard
-                    urlBasePath={urlBasePath}
-                    fetchTelemetry={fetchTelemetry}
-                    onConfirm={this.handleTelemetrySelection.bind(this, true)}
-                    onDecline={this.handleTelemetrySelection.bind(this, false)}
+                <SampleDataCard
+                  urlBasePath={urlBasePath}
+                  onConfirm={this.onSampleDataConfirm}
+                  onDecline={this.onSampleDataDecline}
+                />
+                <EuiSpacer size="s" />
+                <EuiTextColor className="euiText--small" color="subdued">
```

Comment:
This part is a bit hectic, but I couldn't find a better mechanism for embedding links into a formatted message. Would love to know if folks have a better alternative as this is super noisy.